scanapp/tests/test_models.py

```diff
@@ -0,0 +1,260 @@
+#
+# Copyright (c) 2017 nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-server/
+# The scancode-server software is licensed under the Apache License version 2.0.
+# Data generated with scancode-server require an acknowledgment.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with scancode-server or any scancode-server
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with scancode-server and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  scancode-server should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  scancode-server is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-server/ for support and download.
+
+import json
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.utils import timezone
+
+from scanapp.models import Copyright
+from scanapp.models import CopyrightAuthor
+from scanapp.models import CopyrightHolder
+from scanapp.models import CopyrightStatement
+from scanapp.models import License
+from scanapp.models import Package
+from scanapp.models import Scan
+from scanapp.models import ScanError
+from scanapp.models import ScannedFile
+
+
+class ScanTestCase(TestCase):
+    def test_scan_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        
+        self.assertEqual('2.0.0rc3', scan.scancode_version)
```

Comment:
You need to get the object you have saved ... otherwise you are not testing anything